tests/test_observation_models.py

```diff
@@ -0,0 +1,123 @@
+import jax
+import jax.numpy as jnp
+import numpy as np
+import pytest
+import scipy.stats as sts
+import statsmodels.api as sm
+
+import nemos as nmo
+
+
+class TestPoissonObservations:
+    cls = nmo.observation_models.PoissonObservations
```

Comment:
why do this instead of just having `PoissonObservations` in each method or as a fixture? current version isn't particularly transparent (what observation model?) and doesn't save time (which fixture would)